### PR TITLE
add support for dynamic metrics

### PR DESCRIPTION
**Pull request by sandlbn**
Based on influxdb publisher code. 

Tested on local opentsdb.

### CODE REVIEW

**Comment by IRCody** [comment]
cc: @intelsdi-x/plugin-maintainers

File: opentsdb/opentsdb.go [hunk 1]
```diff
@@ -109,6 +109,21 @@ func (p *opentsdbPublisher) Publish(contentType string, content []byte, config m
...
+		if isDynamic {
+			for i, j := range indexes {
+				// The second return value from IsDynamic(), in this case `indexes`, is the index of
+				// the dynamic element in the unmodified namespace. However, here we're deleting
+				// elements, which is problematic when the number of dynamic elements in a namespace is
+				// greater than 1. Therefore, we subtract i (the loop iteration) from j
+				// (the original index) to compensate.
+				//
+				// Remove "data" from the namespace and create a tag for it
+				ns = append(ns[:j-i], ns[j-i+1:]...)
```
Comment (katarzyna-z) [inline opentsdb/opentsdb.go]: I don't see usage of modified namespace, I see that here it is the last usage of `ns`. 
I think that there is a problem in this [line](https://github.com/intelsdi-x/snap-plugin-publisher-opentsdb/pull/17/files#diff-3c60c09c9e11331ea5b853dfd917d51cR135) and in this line modified namespace should be in use.

@sandlbn Could you take a look at this?

**Comment by katarzyna-z** [comment]
Please increment plugin version in [this](https://github.com/intelsdi-x/snap-plugin-publisher-opentsdb/pull/17/files#diff-3c60c09c9e11331ea5b853dfd917d51cR41) line.

**Comment by IRCody** [comment]
@katarzyna-z, @sandlbn: Can we update this PR, it's been open awhile.

cc: @intelsdi-x/plugin-maintainers

File: opentsdb/opentsdb.go (hunk 1 above)
Comment (sandlbn) [inline opentsdb/opentsdb.go]: @katarzyna-z ok done.

**Comment by katarzyna-z** [comment]
Now, only the plugin version should be increased.

**Comment by katarzyna-z** [comment]
LGTM

**Pull request merged**